### DISCUSSION TITLE
Early Universe Quasar

### DISCUSSION THREAD

**Discussion by MJoseph315** [a-cosmology-group/acg#203] [Astrophysics and Physics]
Thoughts on this study?
https://www.space.com/molecular-gas-fountain-ancient-universe-black-hole

**Comment by mikehelland** [comment]
Quasars are just active galactic nuclei with an absolute magnitude of less than -23 or something pretty arbitrary. I made this page to investigate the data (select DR16Q):

https://mikehelland.github.io/hubbles-law/other/sdss.htm

They have a lower magnitude (meaning brighter) in the past than today.

Whenever we scrutinize that data, we usually find they are a double quasar, have intermingling starlight, or are lensed.*

We should be highly suspicious that the brightest ones are actually that bright. Of the 750,000 quasars in the catalog, less than 7000 have abs mag < -28. This selection of the data is largely what their conclusions are based on.

* double quasars and intermingling starlight:  
https://youtu.be/FCLv6W-D5hY?si=nS_-l1EEcwwh3lTr&t=399

* lensed
https://www.youtube.com/watch?v=pJ08d6vIcAw

**Reply by RedshiftDrift** [reply]
... in addition all quasar spectra show a [Lyman-alpha forest](https://pages.astronomy.ua.edu/keel/agn/forest.html), which means that their light has traveled trough a lot of hydrogen gas that amplifies the redshift.  The large distance and apparent luminosity of quasars is an illusion, [quasars are a lot closer than the distance implied by their redshift](http://arxiv.org/abs/astro-ph/0602242).  The study cited above is worthless.

**Reply by MJoseph315** [reply]
When I made this post, I think I did so because it seemingly confirmed some mainstream theories about why mature, red galaxies were showing up in the early universe.
